### PR TITLE
chore: refactor

### PR DESCRIPTION
**Pull request by Hoijof**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,8 @@
   },
   "devDependencies": {
     "concurrently": "^6.1.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^3.4.0",
```
Comment (amatiasq) [inline package.json:36]: Yas

File: src/game/Circle.ts [hunk 2]
```diff
@@ -1,60 +1,41 @@
...
 
-    updateAction(updateAction: number[]) {
+    updateAction(updateAction: CircleBasicUpdate) {
         for (let i = 1; i < updateAction.length; i += 2) {
             this[this.indexToKey[updateAction[i]]] = updateAction[i + 1];
         }
     }
 
-    update(delta): CircleBasicUpdate {
+    update(delta): Array<number> {
```
Comment (amatiasq) [inline src/game/Circle.ts:38]: WAT? weren't you doing the opposite?

File: src/types/gameTypes.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  data: OrderData
+};
+
+
+export interface OrderData {
+  x?: number,
+  y?: number
+}
+
+export type CircleBasicUpdate = FixedSizeArray<3, number>;
```
Comment (amatiasq) [inline src/types/gameTypes.ts:48]:
```ts
export type CircleBasicUpdate = [number, number, number]
```
?

**Review by amatiasq — COMMENTED** [review]
Cartabuena

File: src/game/Circle.ts (hunk 2 above)
Comment (Hoijof) [reply src/game/Circle.ts:38]: This method is defined in GameObject too. :D I don't want GameObject to know about Circle .

**Pull request merged**